framework/src/wifi_manager/wifi_utils.c

```diff
@@ -380,14 +383,14 @@ wifi_utils_result_e wifi_utils_start_softap(wifi_utils_softap_config_s *softap_c
 		ap_config->channel = softap_config->channel;
 	}
 
-	if (softap_config->ssid == NULL) {
+	if (softap_config->ssid_length < 1) {
 		goto start_soft_ap_fail;
 	} else {
 		memcpy(&ap_config->ssid, softap_config->ssid, softap_config->ssid_length);
 		ap_config->ssid_len = softap_config->ssid_length;
 	}
 
-	if (!softap_config->passphrase || !softap_config->passphrase_length) {
+	if (softap_config->passphrase_length < 1) {
 		goto start_soft_ap_fail;
 	} else {
 		security_config = (slsi_security_config_t *)zalloc(sizeof(slsi_security_config_t));
```

Comment:
In this file I can see SLSI macros being used directly. It makes it very specific to chip.

Reply:
Yes, current wifi_manager implementation has dependency on slsi_wifi.
We will introduce some abstraction layers between wifi_manager and slsi_wifi when other wi-fi chipset vendors join Tizen RT.